application/views/admin/dashboard/ideas.php

```diff
@@ -61,7 +60,7 @@
             }
             ?>">
               <td>
-                <a href="<?php echo base_url() . 'home/idea/' . $idea->id . "/" . $freename;?>" target="_blank"><?php echo $idea->title; ?></a>
+                <a href="<?php echo base_url() . 'home/idea/' . $idea->id;?>" target="_blank"><?php echo $idea->title; ?></a>
```

Comment:
Be aware that the name of the idea in the url is for SEO purposes. Please don't remove them.
I suggest you add `// TODO: replace this with sluglify functions ` on line 43 and the others $freename, I'll update them later.